scripts/verify.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+MANIFEST_URL="https://getalby.com/install/hub/manifest.txt"
+SIGNATURE_URL="https://getalby.com/install/hub/manifest.txt.asc"
+
+verify_package() {
+  local archive_file="${1}"
+  local filename_in_manifest="${2}"
+  local response=""
+
+  while true; do
+    read -r -p "Verify package signature and integrity? (Y/N): " response
+    case "$response" in
+      [Yy]) break ;;
+      [Nn]) echo "Verification skipped." ; return 0 ;;
+      *) echo "Invalid input. Please enter Y or N." ;;
+    esac
+  done
+
+  for cmd in gpg sha256sum; do
+    if ! command -v "$cmd" &>/dev/null; then
+      echo "❌ Required command '$cmd' is not available." >&2
+      return 1
+    fi
+  done
+
+  echo "Downloading manifest file..."
+  if ! wget -q "$MANIFEST_URL"; then
+    echo "❌ Failed to download manifest file." >&2
+    return 1
+  fi
+
+  echo "Downloading manifest signature file..."
+  if ! wget -q "$SIGNATURE_URL"; then
+    echo "❌ Failed to download manifest signature file." >&2
+    return 1
+  fi
+
+  if ! gpg --batch --verify "manifest.txt.asc" "manifest.txt"; then
+    echo "❌ GPG signature verification failed!" >&2
```

Comment:
What happens if the user hasn't imported the gpg key yet?

Should we update this message to also include: `. Visit https://github.com/getAlby/hub/releases for more information on how to verify the release`